### ISSUE TITLE
[1.4.4 (33) Live Build] Asus Zenfone Go 4.5 / Android 5.1 - Video length is small - Create lomotif with 3 snapchat videos and make it as 30 sec

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-android-issues#345]
Frequency: Every time

Action:
1. Launch the app
2. Tap on START
3. On Add Photos & Videos screen, add 3 videos from snapchat folder
4. Don't add any song and create lomotif
5. On create screen, click on added first video, it will display video timeline
6. Drag the slider at the end of the video
7. Do the same process for remaining 2 videos
8. Now, Tap adjust tab and change the video time to 30 sec from 15 sec
9. Check the preview, does it play for 30 sec or not

Expected result: Created lomotif preview should play 30 sec

Actual result: Lomotif plays only for 3-4 sec instead 30 sec

**Comment by testershub** [comment]
Not able to reproduce on [1.4.5(34)] for Samsung Galaxy Note 4 / 6.0.1, it is working as expected.

**Comment by testershub** [comment]
Please see, this issue seems OS specific as I am able to reproduce this issue on Samsung Galaxy S 4/5.0.1 but on Samsung Galaxy Note 4/6.0.1 it is working as expected for the latest build  [1.4.5(34)].

**Comment by xiuie** [comment]
@hiteshbheda How long are the original videos? (i.e. added together).. If they are 1 sec each, the maximum video length will be 1 sec x 3.. If they are more than 30secs (when combined) then this is a bug. @testershub

**Comment by hiteshbheda** [comment]
@xiuie @testershub The original video are of length 2 sec, 3 sec and 4 sec respectively. 

I'll try to check this issue again by considering videos more than 30 sec length.

Please check attached video to reproduce this issue:
[Video](https://drive.google.com/drive/u/0/folders/0B6P2U77GNA4UdzhwVG9WaFZ0aVE)

**Comment by testershub** [comment]
@xiuie @hiteshbheda for three Snapchat videos each of length 10 secs (total 30 seconds), the lomotif created with above stated actions steps is of 3-5 seconds.

**Comment by hiteshbheda** [comment]
@xiuie Yes, I am facing the same issue as @testershub is facing as per above comment
